src/components/MeasurementOverlay/MeasurementOverlay.js

```diff
@@ -66,21 +74,49 @@ class MeasurementOverlay extends React.PureComponent {
     core.removeEventListener('annotationChanged', this.onAnnotationChanged);
   }
 
-  onMouseMove = () => {
+  onMouseMove = e => {
     const { activeToolName, openElement } = this.props;
     const tool = core.getTool(activeToolName);
 
     if (this.state.annotation) {
+      const insideRect = this.isMouseInsideRect(e, this.overlayRef.current);
+      let useTransparentBackground;
+
+      if (this.isCreatingAnnotation) {
```

Comment:
This is the main logic to control if we should render the component transparent. Let me know if you have any questions.